stories/NavPrimaryMobile.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import { NavPrimaryMobile, NavProps } from "../components/Nav/NavPrimaryMobile"
+
+export default {
+	title: "Example/Nav",
+	component: NavPrimaryMobile,
+} as ComponentMeta<typeof NavPrimaryMobile>
+
+const Template: ComponentStory<typeof NavPrimaryMobile> = (args) => (
+	<NavPrimaryMobile {...args} />
+)
+
+export const MobilePrimaryNav = Template.bind({})
+
+MobilePrimaryNav.args = {
+	activeNavLink: "home",
```

Comment:
```suggestion
	activeNavLink: "/home",
```

I think I may have written this wrong in the instructions. Can you add a / and see if the active class starts showing up on home?

Reply:
sure, lemme try

Reply:
yes, it worked, pushing the new changes